### ISSUE TITLE
DD3 R4. Include general section on language to use

### ISSUE THREAD

**Issue by PeterParslow** [agiorguk/gemini#35]
Explicitly encourage the use of plain language, referencing for example the UK government writing guidance at https://www.gov.uk/guidance/content-design/writing-for-gov-uk  and the “[Accessibility and assisted digital](https://www.gov.uk/service-manual/helping-people-to-use-your-service)” section of government service manual, in particular their discussion of [WCAG 2.1 design principle 3 Understandable](https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.gov.uk%2Fservice-manual%2Fhelping-people-to-use-your-service%2Funderstanding-wcag&data=02%7C01%7CPeter.Parslow%40os.uk%7Ce1cf575f550d4de6416808d84b352d22%7C7988742dc5434b9a87a910a7b354d289%7C0%7C0%7C637342039056408381&sdata=0rtMs4RbWs21aN%2FMsvXWMOkdJTT7xE%2FcynmOLINEK4k%3D&reserved=0).
Encourage use of terms that users search for, especially in elements _Title_, _Keyword_ and _Abstract_. This may also apply to _Lineage_ and licence information.
For _Extent_, this means using terms that different user communities use for places; this may include both formal place names from controlled lists such as counties, constituencies, or statistical units, and informal names. The metadata creator should be careful to use a name which is a close match for the area being described.

**Comment by archaeogeek** [comment]
I think we've slowly started to address this with recent changes to the guidance for Alternative Title. I would be happy to see similar changes in the guidance for the Abstract, but I think there's an opposing view for all those elements that are generally populated from controlled lists (Keywords, Extents, Licence), namely that allowing free text will cause problems and actually make discovery harder. For Lineage (https://agiorguk.github.io/gemini/1062-gemini-datasets-and-data-series.html#10), I think our guidance is already sufficient?

**Comment by PeterParslow** [comment]
Agree - the recommendation for "plain English" only applies to free text fields

But to improve search performance, choose keywords & place names that are themselves "plain English"!

**Comment by archaeogeek** [comment]
Are we happy to close this one now then @PeterParslow?

**Comment by archaeogeek** [comment]
Confirmed to close
